pkg/clipboard/clipboard_test.go

```diff
@@ -3,21 +3,34 @@ package clipboard
 import (
 	"bytes"
 	"context"
+	"github.com/atotto/clipboard"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/atotto/clipboard"
 	"github.com/gopasspw/gopass/internal/out"
 	ps "github.com/mitchellh/go-ps"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestCopyToClipboard(t *testing.T) {
+func TestNotExistingClipboardCopyCommand(t *testing.T) {
+	_ = os.Setenv("GOPASS_NO_NOTIFY", "true")
+	_ = os.Setenv("GOPASS_CLIPBOARD_COPY_CMD", "not_existing_command")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	buf := &bytes.Buffer{}
+	out.Stdout = buf
+
+	assert.Error(t, CopyTo(ctx, "foo", []byte("bar"), 1))
+}
+
+func TestUnsupportedCopyToClipboard(t *testing.T) {
 	_ = os.Setenv("GOPASS_NO_NOTIFY", "true")
+	_ = os.Setenv("GOPASS_CLIPBOARD_COPY_CMD", "")
```

Comment:
Since we might set the environment variable in a previous test we need to "unset" it.